controller/src/controller.rs

```diff
@@ -43,7 +43,7 @@ impl<T: BottlerocketNodeClient> BrupopController<T> {
             .iter()
             .filter(|brn| {
                 brn.status.as_ref().map_or(false, |brn_status| {
```

Comment:
Seems `map_or_else` would be more suitable here?

Reply:
I chatted with @somnusfish about this elsewhere. To summarize, we would only want `map_or_else` if the default value is something that needs to be computed.